language/English/strings.po

```diff
@@ -11579,7 +11579,43 @@ msgctxt "#21470"
 msgid "%s [%d]"
 msgstr ""
 
-#empty strings from id 21471 to 21601
+#: system/settings/settings.xml
+msgctxt "#21471"
+msgid "Initially"
+msgstr ""
```

Comment:
@HitcherUK explained it much better and more concisely than I did.

@MartijnKaijser where should the descriptions be placed? Are there existing examples so I can see how things should be?